.ci/azure-pipelines/build_windows.bat

```diff
@@ -60,7 +61,9 @@ if errorlevel 1 (
 	exit /b %errorlevel%
 )
 
-ctest
+:: Set timeout for tests to 10min
+SET QTEST_FUNCTION_TIMEOUT=600000
```

Comment:
I would write the command in lowercase, as that's the style used in the rest of the file.

Reply:
Sure - I always wrote commands in all caps but you are right. It's not how it's done in the rest of the file.

Feel free to change it yourself. If oxi don't do it I'll probably do it some time tomorrow or so :)

Reply:
Just realized `set` was written in uppercase above. I fixed it.